### PR TITLE
Fixes RSD route in ArticulateRouter

### PR DESCRIPTION
**Pull request by warrenbuckley**
Helps get one more thing done for getting this upto date for V5 https://github.com/Shazwazza/Articulate/issues/381

### CODE REVIEW

File: src/Articulate/Routing/ArticulateRouter.cs [hunk 1]
```diff
@@ -105,6 +106,7 @@ public void MapRoutes(HttpContext httpContext, IUmbracoContext umbracoContext)
                     //MapMetaWeblogRoute(routes, uriPath, articulateRootNode);
                     //MapManifestRoute(routes, uriPath, articulateRootNode);
                     //MapRsdRoute(routes, uriPath, articulateRootNode);
```
Comment (Shazwazza) [inline src/Articulate/Routing/ArticulateRouter.cs]: Suppose this line can be removed?

**Review by Shazwazza — COMMENTED** [review]
I don't have much time to test but I assume this follows the other routing patterns and it works?

**Comment by Shazwazza** [comment]
Once conflicts resolved i can merge

**Comment by warrenbuckley** [comment]
@Shazwazza Conflicts fixed
image

**Pull request merged**